Lib/test/test_zstd/test_core.py

```diff
@@ -0,0 +1,2506 @@
+import array
+import gc
+import io
+import pathlib
+import pickle
+import random
+import builtins
+import re
+import os
+import unittest
+import tempfile
+import threading
+
+from compression._common import _streams
+
+from test.support.import_helper import import_module
+from test.support import threading_helper
+from test.support import _1M
+from test.support import Py_GIL_DISABLED
+
+zstd = import_module("compression.zstd")
+_zstd = import_module("_zstd")
```

Comment:
Is it good practice to test it there? Should the related tests be moved from `Lib` to `Modules`?

Reply:
Yeah the tests all live in `Lib/tests` except for `[_testcapi](https://github.com/python/cpython/tree/main/Modules/_testcapi), [_testinternalcapi](https://github.com/python/cpython/tree/main/Modules/_testinternalcapi), and [_testlimitedcapi](https://github.com/python/cpython/tree/main/Modules/_testlimitedcapi). At least as far as I am aware. I should probably move this import_module call first however.